src/JSON.jl

```diff
@@ -1,7 +1,9 @@
-__precompile__()
+# __precompile__() rene
```

Comment:
?

Reply:
sorry about that, had commented it out for quicker development cycles.